infrastructure/models.py

```diff
@@ -308,6 +337,54 @@ def fuzzy_output_date(self):
     estimated_transfer_capacity = models.BigIntegerField(
         null=True, blank=True, help_text="Estimated Transfer Capacity (MW)"
     )
+    # ---- END TRANSMISSION TYPE PROJECT FIELDS ---- #
+
+    # ---- PIPELINE TYPE PROJECT FIELDS ---- #
+    pipeline_diameter = models.IntegerField("Pipeline Diameter", null=True, blank=True)
+    pipeline_diameter_unit = models.CharField("Diameter Unit", null=True, blank=True, max_length=6,
```

Comment:
Ideally, we don't need ``null=True`` on a ``CharField``, ``blank=True`` should be enough.